packages/alchemy/src/config/utils/cookies.ts

```diff
@@ -0,0 +1,83 @@
+import { getCookie, removeCookie, setCookie } from "typescript-cookie";
+import { DEFAULT_SESSION_MS } from "../../signer/session/manager.js";
+import type { AlchemyAccountsConfig, AlchemyClientState } from "../types";
+import { deserialize } from "./deserialize.js";
+
+/**
+ * Function to create cookie based Storage
+ *
+ * @param config optional config object that allows you to set the session length
+ * @returns an instance of a browser storage object that leverages cookies
+ */
+export const cookieStorage: (config?: { sessionLength: number }) => Storage = (
+  config
+) => ({
+  // this is unused for now, we should update this if we do need it
+  length: 0,
+
+  clear: function (): void {
+    if (typeof document === "undefined") return;
+
+    document.cookie = "";
```

Comment:
This feels sketchy. The app using this library might be using cookies for its own storage unrelated to aa-sdk, and wouldn't expect aa-sdk to delete all of their cookies under any circumstances.

I'm not sure if this is actually called anywhere in the sdk. If it's not, I might remove `clear` from this interface. Alternatively, we could have a prefix for all keys added by aa-sdk and have clear delete all the keys with that prefix.

Reply:
Oops, duplicated comment.